README.md

```diff
@@ -53,7 +56,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(1). This is because you will only need two pointers traversing either end, no extra memory is needed.
```

Comment:
It you have n tiles, then you need to keep at least half of them in memory, so the algorithm will be O(n).

# -3